api/views.py

```diff
@@ -0,0 +1,96 @@
+from django.http import JsonResponse
+from rest_framework import viewsets
+from products.models import Products, Category
+from stockmgmt.models import StockHistory
+from .serializer import ProductsSerializer, CategorySerializer, StockHistorySerializer
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.response import Response
+
+
+
+@api_view(['GET', 'POST'])
+def products_list(request,format=None):
```

Comment:
Please use class based views.

Reply:
so products_get, products_post and etc??